frappe_graphql/utils/resolver/document_resolver.py

```diff
@@ -0,0 +1,51 @@
+from graphql import GraphQLResolveInfo
+
+import frappe
+from frappe.model import default_fields
+
+from .utils import get_singular_doctype
+
+
+def document_resolver(obj, info: GraphQLResolveInfo, **kwargs):
+    doctype = obj.get('doctype') or get_singular_doctype(info.parent_type.name)
+    if not doctype:
+        return None
+
```

Comment:
add pr #12 here ? seems like custom types aren't being resolved.
   ```
 # check if requested field can be resolved
    if isinstance(obj, dict):
        resolved_field_name = (obj or {}).get(info.field_name)
        if resolved_field_name:
            return resolved_field_name
```
That works